CHANGELOG.md

```diff
@@ -243,6 +243,6 @@ _October 21th, 2019_
 - `ChannelPresenter.channelDidUpdate` observable (for example to get updated members).
 - `ChannelsViewController` UI warnings. It tries to update itself when it's not in the hierarchy view.
 
-##### Breaking changes
+## Breaking changes
```

Comment:
Here should be 3 `###`. I described the flow :)

Reply:
I thought you said "Breaking changes" should be ## and all else should be ###?